projects/angular/src/forms/common/label.ts

```diff
@@ -32,6 +42,18 @@ export class ClrLabel implements OnInit, OnDestroy {
     return this.el.nativeElement && this.el.nativeElement.textContent;
   }
 
+  /**
+   * Allowing signposts inside labels to work without disabling default behavior. <label> is spreading a click event to its children so signposts get
+   * automatically closed once clicked inside a <label>.
+   * @param event
+   */
+  @HostListener('click', ['$event'])
+  onClick(event) {
+    if (event.target.hasAttribute('clrSignpostTrigger')) {
```

Comment:
Can you add unit tests to validate both branches here (if and else)?

Reply:
Added in [f9c41c0](https://github.com/vmware-clarity/ng-clarity/pull/937/commits/f9c41c03bf702361fc333db07b6d057093858f31)

Reply:
This doesn't catch all instances and leads to some odd behavior. 

 - Clicking inside the signpost when opened will cause it to close. The signpost should be able to receive focus and stay open. 
 - The events from within the signpost (close button, click in signpost-content) still bubble up to the label and are not defaulted.

Reply:
I added a more strict check to prevent that. Actually the `click` events were still coming from the `label` due to the condition not being met. So when you click inside the `signpost` it's again in the `label` and label is spreading that to the `button` inside `signpost` which is closing it.

Reply:
Much better UX now. Thanks for updating.